.github/ServiceX/scripts/proxy-exporter.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+#sudo mkdir -p /etc/grid-security
```

Comment:
delete commented-out code

Reply:
That is there on purpose. It will be needed eventually, but for this first test @mweinberg2718 and I are sidestepping anything having to do with `sudo` or grid proxies.